### PR TITLE
charts: service: move loadBalancerIP from port to top level of spec

### PR DESCRIPTION
**Pull request by BinaryMan32**
The `loadBalancerIP` setting should be at the same level as
`type: LoadBalancer`, not in the per-port settings.

### CODE REVIEW

**Comment by morganchristiansson** [comment]
This is correct. Apologies - I added it with wrong indentation in my previous PR. LGTM.

**Comment by BinaryMan32** [comment]
@morganchristiansson I added another commit to bump the chart version to 1.0.2. Anything else I'd need to dp before this PR can be merged?

**Comment by morganchristiansson** [comment]
Looks good to me. @networkop can you merge

**Comment by Marx2** [comment]
who can push it?

**Comment by morganchristiansson** [comment]
I had a look. I made chart 1.0.2 available from helm repo url https://morganchristiansson.github.io/k8s_gateway

After running `make helm-package` to update index.yaml and enabled GitHub pages from master on my fork https://github.com/morganchristiansson/k8s_gateway.

**Comment by networkop** [comment]
thanks @morganchristiansson  and @BinaryMan32 . I am no longer part of the ori-edge org, but I've asked to be added to the list of maintainers (see #23 ). I'll try to ping people offline to see if it's possible. If not, I suspect we may have to fork this and move the active maintenance elsewhere.

**Comment by morganchristiansson** [comment]
Forking would be easy if ori edge can add note in readme. Can even rename/move repo and github will redirect to new org/user&name.

**Review by networkop — APPROVED** [review]

**Comment by networkop** [comment]
all good @morganchristiansson , I've been added to the list of maintainers.

**Pull request merged**
